tools/get_coverage_ros2_pkg.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import requests
+import subprocess
+import sys
+import tempfile
+
+parser = argparse.ArgumentParser()
```

Comment:
Best practice recommends to move all lines which should be run when the file is invoked into a conditional block: `if __name__ == '__main__':`

That allows to import the file without executing the logic.

Reply:
a507211 and c6cebed